modin/pandas/groupby.py

```diff
@@ -180,18 +186,17 @@ def idxmax(self):
     @property
     def ndim(self):
         """
-        Return 2.
+        Return the number of dimensions of the source DataFrame.
 
         Returns
         -------
         int
-            Returns 2.
 
         Notes
         -----
         Deprecated and removed in pandas and will be likely removed in Modin.
         """
-        return 2  # ndim is always 2 for DataFrames
+        return self._ndim
```

Comment:
as it's revealed `ndim` isn't always 2 for DataFrameGroupBy
```python
>>> grp = pandas.DataFrame({"a": [1, 2, 2, 3], "b": [3, 4, 5, 6]}).groupby("a", as_index=False)["b"]
>>> type(grp)
<class 'pandas.core.groupby.generic.DataFrameGroupBy'>
>>> grp.ndim
1
```

Reply:
[`ndim`](https://github.com/pandas-dev/pandas/blob/5f648bf1706dd75a9ca0d29f26eadfbb595fe52b/pandas/core/base.py#L201-L202) of pandas GroupBy object is the `ndim` of the [`_selected_obj`](https://github.com/pandas-dev/pandas/blob/098661e8773dc2026f89ec39ca87625c2fb870dd/pandas/core/groupby/groupby.py#L681-L689) of this GroupBy. When `key` for the selection is list-like, then the selected object (which is `source_df[key]`) will always be a Series (its `ndim` is 1). 

This rule looks consistent, not sure that this is some kind of issue

Reply:
removed note about deprecation